src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
you can add horizontal 50px paddings for header and get rid of icon's left  and nav's right margins